api/acl.go

```diff
@@ -72,9 +72,31 @@ func (c *Client) ACLTokens() *ACLTokens {
 	return &ACLTokens{client: c}
 }
 
-// Bootstrap is used to get the initial bootstrap token
-func (a *ACLTokens) Bootstrap(q *WriteOptions) (*ACLToken, *WriteMeta, error) {
```

Comment:
I missed this the first time through, but changing the signature here breaks backwards compatibility for the Nomad Go API (used by Terraform and third-parties) even though the HTTP API change has been done in a backwards compatible way. So rather than changing the `Bootstrap` method signature, we'll need to:

* Add a new method named `BootstrapOpts` (I'm 😐  about the name but that aligns with other examples)
* Add a comment to the old `Bootstrap` method indicating that the old method is deprecated and will be removed in Nomad 1.5.0.
* Update the code you've changed in this PR to use the new method.

Reply:
I have implemented this in the latest push